setup.cfg

```diff
@@ -0,0 +1,158 @@
+# Discord educational bot for the Aerospace Engineering Python course
+# Copyright (C) 2020 Delft University of Technology
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public
+# License along with this program.
+# If not, see <https://www.gnu.org/licenses/>.
+
+[metadata]
+name = edubot
+description = Discord bot for the TU Delft Aerospace Engineering Python course
+long_description = file: README.md
+long_description_content_type = text/markdown
+keywords =
+    discord
+    bot
+    python
+    online-lecturing
+license = GNU Affero General Public License v3
+license-file = LICENSE
+url = https://github.com/TUDelft-CNS-ATM/EduBot
+author = Joost Ellerbroek
+author_email = j.ellerbroek@tudelft.nl
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Education
+    License :: OSI Approved :: GNU Affero General Public License v3
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Communications :: Chat
+    Topic :: Education
```

Comment:
@jooste What do you think of the corrected metadata? There are a list of available classifiers here: https://pypi.org/classifiers/

Reply:
Looks good!